components/template/Templates.vue

```diff
@@ -73,7 +73,7 @@ export default {
   props: {
     activeCategory: Object,
     templates: Array,
-    templateCategories: Array,
+    templateCategories: Object,
```

Comment:
How it is working last time?

Reply:
It was giving data type mismatch error. `Expected an Array got an Object`

Reply:
Okay, Make sure we are updating the right data type.